lib/checks/keyboard/focusable-no-name-evaluate.js

```diff
@@ -1,9 +1,11 @@
 import { isFocusable } from '../../commons/dom';
 import { accessibleTextVirtual } from '../../commons/text';
+import { parseTabindex } from '../../core/utils';
 
 function focusableNoNameEvaluate(node, options, virtualNode) {
-  const tabIndex = virtualNode.attr('tabindex');
-  const inFocusOrder = isFocusable(virtualNode) && tabIndex > -1;
+  const tabIndex = parseTabindex(virtualNode.attr('tabindex'));
+  const inFocusOrder =
+    isFocusable(virtualNode) && tabIndex !== null && tabIndex > -1;
   if (!inFocusOrder) {
```

Comment:
This is essentially just `isInTabOrder`. Let's use that.

```suggestion
  if (!isInTabOrder(virtualNode)) {
```